<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FCurveDlg</class>
 <widget class="QDialog" name="FCurveDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>508</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>F-Curve</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_11">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeView" name="channelView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QTreeView
{
	background-color: rgb(43,43,43);
	show-decoration-selected: 1;
	font: 12pt 'HarmonyOS Sans';
}

QTreeView::item {
	color: #858280;
	border: 1px solid transparent;
}

QTreeView::item:hover {
	background-color: transparent;
}

QTreeView::item:selected {
	border: 1px solid #4B9EF4;
	background: #334960;
	color: #ffffff;
}

QTreeView::item:selected:active{
	background: #334960;
	color: #ffffff;
}

QTreeView::item:selected:!active {
	background: #334960;
	color: #ffffff;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="CurveMapView" name="gridview">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="topMargin">
        <number>19</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item row="0" column="5">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="spacing">
          <number>16</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">QLabel
{
	font: 14px 'HarmonyOS Sans';
	color: rgb(133, 130, 128);
}</string>
           </property>
           <property name="text">
            <string>Tangent    Left X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editTanLeftX">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="styleSheet">
            <string notr="true">QLabel
{
	font: 14px 'HarmonyOS Sans';
	color: rgb(133, 130, 128);
}</string>
           </property>
           <property name="text">
            <string>Right X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editTanRightX">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="5">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="spacing">
          <number>16</number>
         </property>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="styleSheet">
            <string notr="true">QLabel
{
	font: 14px 'HarmonyOS Sans';
	color: rgb(133, 130, 128);
}</string>
           </property>
           <property name="text">
            <string>Tangent    Left Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editTanLeftY">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="styleSheet">
            <string notr="true">QLabel
{
	font: 14px 'HarmonyOS Sans';
	color: rgb(133, 130, 128);
}</string>
           </property>
           <property name="text">
            <string>Right Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editTanRightY">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">QLabel
{
	font: 14px 'HarmonyOS Sans';
	color: rgb(133, 130, 128);
}</string>
         </property>
         <property name="text">
          <string>Point X</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="btnFree">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnVector">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAligned">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAsymmetry">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">QLabel
{
	font: 14px 'HarmonyOS Sans';
	color: rgb(133, 130, 128);
}</string>
         </property>
         <property name="text">
          <string>Point Y</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">QLabel
{
	font: 14px 'HarmonyOS Sans';
	color: rgb(133, 130, 128);
}</string>
         </property>
         <property name="text">
          <string>Handle type</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="btnLoadPreset">
           <property name="text">
            <string>Load Preset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSavePreset">
           <property name="text">
            <string>Save Preset</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="3">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="editPtX">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ZIconLabel" name="btnLockX"/>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true">QLabel
{
	font: 14px 'HarmonyOS Sans';
	color: rgb(133, 130, 128);
}</string>
           </property>
           <property name="text">
            <string>Frame</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editFrame">
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="3">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLineEdit" name="editPtY">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ZIconLabel" name="btnLockY"/>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">QLabel
{
	font: 14px 'HarmonyOS Sans';
	color: rgb(133, 130, 128);
}</string>
           </property>
           <property name="text">
            <string>Value</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editValue">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">QLabel
{
	font: 14px 'HarmonyOS Sans';
	color: rgb(133, 130, 128);
}</string>
           </property>
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="editXFrom">
             <property name="minimumSize">
              <size>
               <width>36</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame">
             <property name="minimumSize">
              <size>
               <width>5</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>5</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="palette">
              <palette>
               <active>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>138</red>
                   <green>138</green>
                   <blue>140</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>138</red>
                   <green>138</green>
                   <blue>140</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>120</red>
                   <green>120</green>
                   <blue>120</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="frameShape">
              <enum>QFrame::HLine</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editXTo">
             <property name="minimumSize">
              <size>
               <width>36</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">QLabel
{
	font: 14px 'HarmonyOS Sans';
	color: rgb(133, 130, 128);
}</string>
           </property>
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="editYFrom">
             <property name="minimumSize">
              <size>
               <width>36</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="minimumSize">
              <size>
               <width>5</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>5</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="palette">
              <palette>
               <active>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>138</red>
                   <green>138</green>
                   <blue>140</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>138</red>
                   <green>138</green>
                   <blue>140</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>120</red>
                   <green>120</green>
                   <blue>120</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="frameShape">
              <enum>QFrame::HLine</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editYTo">
             <property name="minimumSize">
              <size>
               <width>36</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>64</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>64</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="cbIsTimeline">
         <property name="styleSheet">
          <string notr="true">QCheckBox {
    spacing: 8px;
	font: 14px &quot;HarmonyOS Sans&quot;;
	color: rgba(255, 255, 255, 0.7);
}

QCheckBox::indicator {
    width: 15px;
    height: 15px;
}

QCheckBox::indicator:unchecked {
    image: url(:/icons/checkbox_unchecked.svg);
}

QCheckBox::indicator:checked {
    image: url(:/icons/checkbox_checked_normal.svg);
}</string>
         </property>
         <property name="text">
          <string>timeline</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZIconLabel</class>
   <extends>QLabel</extends>
   <header>zenoui/comctrl/zlabel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CurveMapView</class>
   <extends>QGraphicsView</extends>
   <header>curvemap/curvemapview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
